.enb/make.js

```diff
@@ -55,13 +57,17 @@ module.exports = function(config) {
             [techs.htmlFromBemjson],
 
             // js
-            [techs.browserJs, {
-                target: '?.js',
-                includeYM: true
+            [ techs.babel, {
+                target: '?.es5.js'
```

Comment:
For ES6 classer https://github.com/bem/bem-xjst/pull/238/commits/f29434d56022a8d75c8b4115c40a1954727c0aa0#diff-4fe9281de224b8508c7ebda7e35beadfR123